index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Portfolio Site</title>
+		<link rel='stylesheet' type='text/css' href='styles.css'>
+		<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300&display=swap" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Work+Sans:900&display=swap" rel="stylesheet">
+	</head>
+	<body>
+		<nav>
+			<div class='logo'>
+				<h1>LOGO</h1>
+				<h1 class='logoMobile'></h1>
+			</div>
+			<ul>
+				<li>Home</li>
+				<li>Portfolio</li>
+				<li>About Me</li>
+				<li>Contact</li>
+			</ul>
+			<img src='list-menu.png'/>
+		</nav>
+		
+		<header>
+			<div class='content'>
+				<h1>Andrew Swan</h1>
+				<h2>fullStack_developer</h1>
```

Comment:
When shrinking the browser window to mobile size, this text gets lost

Reply:
seems fine to me - may be fixed once the h2 and h3 tags are updated